frontend/common/types/responses.ts

```diff
@@ -61,9 +61,10 @@ export type AuditLogItem = {
   environment?: Environment;
   project: ProjectSummary;
   related_object_id: number;
-  related_object_type: string;
+  related_object_type: "FEATURE" | "FEATURE_STATE" | "ENVIRONMENT" | "CHANGE_REQUEST";
```

Comment:
This isn't actually the full list, it's missing `"SEGMENT"`, although we may add new ones in the future. 

https://github.com/Flagsmith/flagsmith/blob/main/api/audit/related_object_type.py

Reply:
Cool, will add. I was considering using this info for display but I don't think I will.

Reply:
Not sure if you haven't pushed yet or I just didn't understand the above but `"SEGMENT"` is still missing here?